ci/mkpipeline.py

```diff
@@ -271,11 +271,17 @@ def prioritize_pipeline(pipeline: Any) -> None:
     """Prioritize builds against main or release branches"""
 
     tag = os.environ["BUILDKITE_TAG"]
+    branch = os.environ["BUILDKITE_BRANCH"]
```

Comment:
Minor: Not sure if `BUILDKITE_BRANCH` is always set (when manually triggering a build, which is admittedly seldom in the tests pipeline).